README.md

```diff
@@ -229,6 +232,10 @@ node.init({ emptyRepo: true, bits: 2048 }, (err) => {
 
 > We are working on making this init process better, see https://github.com/ipfs/js-ipfs/issues/556 for the discussion.
 
+More examples can be found in the [examples folder](./examples)
+
+> If you have built an example, please share it with the community by submitting a Pull Request to this repo!.
```

Comment:
Let's remove this line, we don't necessarily want to have thousands of examples, instead we want to have a selected few that cover it all